ostd/src/mm/stat/mod.rs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Statistics prepared for `/proc/meminfo`.
```

Comment:
Don't refer to `/proc/meminfo` in OSTD API documentation. Just say that this module contains APIs for memory statistics.